src/paxcounter_orig.conf

```diff
@@ -25,11 +25,11 @@
 #define BLESCANINTERVAL                 80      // [illiseconds] scan interval, see below, 3 .. 10240, default 80ms = 100% duty cycle
 
 // Corona Exposure Notification Service(ENS) counter
-#define COUNT_ENS                       0       // count found number of devices which advertise Exposure Notification Service
+#define COUNT_ENS                      1       // count found number of devices which advertise Exposure Notification Service
                                                 // set to 0 if you do not want to enable this function
 
 // for additional sensors (added by some user)
-#define HAS_SENSOR_1                    0       // set to 1 if you want to transmit CWA counter
```

Comment:
Do we want this as default? [suggestion: no]

Reply:
remade